packages/blade/src/components/Skeleton/_decisions/decisions.md

```diff
@@ -0,0 +1,327 @@
+  </Box>
+  <Box flexDirection="column" display="flex" gap="spacing.2">
+    <Skeleton />
+    <Skeleton />
+  </Box>
+</Box>
+```
+
+<img src="./skeleton-usage-demo-2.png" alt="Skeleton complex usage" width="50%" />
+
+
+### Working with existing components. 
+
+To use Skeleton loader with existing blade components like Card, consumers can create custom skeleton templates as per their needs and usecases. 
+
+**Example:** 
+
+If the whole card needs to be loadable: 
+
+```jsx
+const App = () => {
+  const { data, isLoading } = useQuery();
+
+  return (
+    <>
+      {isLoading ? (
+        <Box>
+          <Box display="flex">
+            <Box>
+              <Skeleton width="40%" height="30px" />
+              <Skeleton width="30%" height="20px" />
+            </Box>
+            <Skeleton width="30px" height="50px" />
+          </Box>
+          <Skeleton width="100%" height="100px" />
+        </Box>
+      ) : (
+        <Card>
+          <CardHeader>
+            <CardHeaderLeading title={data.title} subtitle={data.subtitle} />
+            <CardHeaderTrailing
+              visual={<CardHeaderBadge variant="neutral">{data.badge}</CardHeaderBadge>}
+            />
+          </CardHeader>
+          <CardBody>
+            <Text>{data.content}</Text>
+          </CardBody>
+        </Card>
+      )}
+    </>
+  );
+};
+```
+
+If only body of the card needs to be lodable consumers can put the skeleton on the CardBody:
+
+```jsx
+const App = () => {
+  const { data, isLoading } = useQuery();
+
+  return (
+    <Card>
+      <CardHeader>
+        <CardHeaderLeading title="Payment options" subtitle="Secured by Razorpay" />
+        <CardHeaderTrailing visual={<CardHeaderBadge variant="neutral">PENDING</CardHeaderBadge>} />
+      </CardHeader>
+      <CardBody>
+        {isLoading ? <Skeleton width="100%" height="100px" /> : <Text>{data.content}</Text>}
+      </CardBody>
+    </Card>
+  );
+};
+
+```
+
+A question that might arise is "Why not just give isLoading prop in the Card?"
+
+Like: 
+
+```jsx
+<Card isLoading={isLoading}>
+  <CardHeader>
+    <CardHeaderLeading title={data.title} subtitle={data.subtitle} />
+    <CardHeaderTrailing visual={<CardHeaderBadge variant="neutral">{data.badge}</CardHeaderBadge>} />
+  </CardHeader>
+  <CardBody>
+    <Text>{data.content}</Text>
+  </CardBody>
+</Card>
+```
+
+We discussed this, while it might look simple this approach also has few downsides: 
+
+- if we provide `isLoading` prop it might cause a lot of jumps and shifts on the page because we won't know until API responds if the CardHeader has title/prefix or subtitle set or not and it could be removed after the API is done loading.
+- Providing `isLoading` prop for all Blade components (where applicable) might not be the most flexible approach.
+- With `isLoading` prop consumers will anyways have to opt out and use custom Skeleton templates for the CardBody content so there's not much benefit to providing it out of the box.
```

Comment:
make it more understandable? the context is missing what exactly we are talking about here?

Reply:
Context is there, if you scroll up you have a codeblock with <Card isLoading \/> prop example, that's what we are talking about here. 

I've made the `A question that might arise is "Why not just give isLoading prop for Card?` bold though, maybe you missed it and didn't get what we are talking about.

> Open to suggestion on wording changes

Reply:
okay here's my proposal:
> With `isLoading` prop consumers will anyways have to opt out and use custom Skeleton templates for the CardBody because CardBody accepts any JSX, so there's not much benefit to providing it out of the box because people will need to tweak it for different use cases

Reply:
Updated.